### PR TITLE
Devtools mv3 samples

### PR DESCRIPTION
**Pull request by IanStanion-google**

### CODE REVIEW

File: api-samples/devtools/panels/chrome-query/devtools.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2012 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
```
Comment (oliverdunk) [inline api-samples/devtools/panels/chrome-query/devtools.js:3]: We should update this to the Apache license :)

File: api-samples/devtools/panels/chrome-query/devtools.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+// The function below is executed in the context of the inspected page.
+/*global $0*/
+let page_getProperties = function () {
+  let data = window.jQuery && $0 ? jQuery.data($0) : {};
```
Comment (oliverdunk) [inline api-samples/devtools/panels/chrome-query/devtools.js:8]: We should probably do a pass updating the code style where we can (e.g, we could use const here).

**Review by oliverdunk — CHANGES_REQUESTED** [review]
Thanks for this! Looks like a great start :)

**Review by oliverdunk — CHANGES_REQUESTED** [review]
Thanks for this! Looks like a great start :)

**Comment by IanStanion-google** [comment]
@oliverdunk Requested changes are in! Let me know if there are any other idioms in the code that should be different that I might have missed.

File: api-samples/devtools/chrome-query/chrome-query.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# devtools
+
+This sample demonstrates using the devtools API to add a sidebar that displays the jQuery data associated with the selected DOM element.
+
+## Overview
+
+The devtools window can be expanded with new elements created by the user. The extension contains html and js files that are injected into the devtools page to expand its functionality.
```
Comment (jpmedley) [inline api-samples/devtools/chrome-query/chrome-query.md:7]:
```suggestion
This extension shows how DevTools can be expanded with new elements. It contains HTML and JavaScript files that are injected into the DevTools page to expand its functionality.
```

**Review by jpmedley — CHANGES_REQUESTED** [review]

File: api-samples/devtools/chrome-query/chrome-query.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
+# devtools
```
Comment (sebastianbenz) [inline api-samples/devtools/chrome-query/chrome-query.md:1]: Should this file be named readme.md?

File: api-samples/devtools/chrome-query/chrome-query.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+
+This sample demonstrates using the devtools API to add a sidebar that displays the jQuery data associated with the selected DOM element.
```
Comment (sebastianbenz) [inline api-samples/devtools/chrome-query/chrome-query.md:3]: Is it still worth demoing anything related to JQuery?

File: api-samples/devtools/panels/chrome-query/chrome-query.md [hunk 6]
```diff
@@ -0,0 +1,7 @@
+# devtools
```
[the remark this answers is not in the document]
Comment (oliverdunk) [reply api-samples/devtools/panels/chrome-query/chrome-query.md]: Thanks for moving this, although can we rename it to panels? chrome-query doesn't make much sense IMO.

File: api-samples/devtools/chrome-query/chrome-query.md (hunk 5 above)
Comment (oliverdunk) [reply api-samples/devtools/chrome-query/chrome-query.md:3]: Long term I think a sample that doesn't use jQuery would be nice, but jQuery still exists, and so I don't think we necessarily need to change this given that our main goal is just having something that works in MV3. This feels like a useful sample for developers migrating even if the context is a bit less common these days.

File: api-samples/devtools/chrome-query/chrome-query.md (hunk 4 above)
Comment (sebastianbenz) [reply api-samples/devtools/chrome-query/chrome-query.md]: Sorry, my mistake, the naming convention is README.md, can you please update the name agian.

**Review by oliverdunk — APPROVED** [review]
LGTM 👍

**Pull request merged**